package.json

```diff
@@ -178,7 +150,7 @@
 		"build:packages": "cross-env EXCLUDE_PACKAGES=babel-plugin-import-jsx-pragma node ./bin/packages/build.js",
 		"build": "npm run build:packages && cross-env NODE_ENV=production webpack",
 		"check-engines": "check-node-version --package",
-		"ci": "concurrently \"npm run lint && npm run build\" \"npm run test-unit:coverage-ci\"",
+		"ci": "concurrently \"npm run lint\" \"npm run test-unit:coverage-ci\"",
```

Comment:
`npm run build` may accidentally regenerate `jest-console` build files which are used in framework setup script for each test suite. We run it on the node which runs e2e tests anyways.

Reply:
I'm having a hard time tracking down why we had `npm run build` here in the first place. If I were to guess, seems as though it were to test to ensure that we'd have no errors during a build. Could still be useful, though I'm not entirely sure it's critical (at least weighted against the time cost / complications).